packages/android_intent/lib/android_intent.dart

```diff
@@ -27,6 +27,8 @@ class AndroidIntent {
   /// [package] refers to the package parameter of the intent, can be null.
   /// [componentName] refers to the component name of the intent, can be null.
   /// If not null, then [package] but also be provided.
+  /// [ignoredPackages] is the list of package names that should not by displayed in the selection dialog and
+  /// should not be used to resolve this intent.
```

Comment:
I think it would also be worth saying this is only used in the `showChooser` method.

Reply:
The `ignoredPackages` argument is used in the `launch` method, if the package is not set.